documentation/modules/auxiliary/scanner/http/meteocontrol_weblog_extractadmin.md

```diff
@@ -0,0 +1,54 @@
+Meteocontrol WEB'Log Data Loggers are affected with an authentication bypass vulnerability. The module exploits this vulnerability to remotely extract Administrator password for the device management portal.
```

Comment:
@h00die Valid point. However, I do not have info on the sw/firmware versions (vuln & fix). The device mgmt portal do not present this info anywhere. Fwiw, I have now added the fix release info in the module to address this.